tests/mock_crust/churn.rs

```diff
@@ -181,20 +219,26 @@ fn request_during_churn_node_to_group() {
         let data_id = data.identifier();
         let message_id = MessageId::new();
 
+        let except_index = added_index.unwrap_or(nodes.len());
+        let quorum_before = quorum(nodes.iter()
+            .enumerate()
+            .filter(|&(i, n)|
+                i != except_index && n.routing_table().is_recipient(&dst.to_destination()))
```

Comment:
I actually think we should be able to just use
```rust
quorum(target_group(&nodes, dst).len())
```
like in the other tests. If the joining node actually ended up in that group and didn't receive the message, all the others should have, and we should still be above quorum. (Also below.)

Since we're not simulating any failing connections in addition, this should never fail.